ui/src/components/breakable-string.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { Fragment } from 'react';
+
+import { cn } from '@/lib/utils';
+
+type BreakableStringProps = {
+  text: string;
+  className?: string;
+};
+
+export const BreakableString = ({ text, className }: BreakableStringProps) => {
+  const parts = text.split(/(:+)/g); // Split by ':' or '::', keeping delimiters
+
+  return (
+    <span className={cn('break-words', className)}>
+      {parts.map((part, index) => (
+        <Fragment key={index}>
+          {part}
+          {part.startsWith(':') && <wbr />}
```

Comment:
If the part starts with ":", should the `<wbr/>` not be before that part?
IIUC the current implementation would create `type:namespace<wbr/>:name<wbr/>:version<wbr/>` instead of `type<wbr/>:namespace<wbr/>:name<wbr/>:version`.
And out of ignorance, if each part is in a separate `Fragment`, is `<wbr/>` even required?

Reply:
Ok, I think I got it, the ":" are individual parts, right? So the result of the split in your example is an array with the elements "Gem", "::", "google-apis-iamcredentials_v1", ":", "0.17.0", correct?